blog/2019-12-18-debezium-1-0-0-final-released.adoc

```diff
@@ -0,0 +1,140 @@
+= Streaming Now: Debezium 1.0 Final Is Out!
+gmorling
+:awestruct-tags: [ releases, mysql, postgres, sqlserver, mongodb, cassandra, oracle ]
+:awestruct-layout: blog-post
+
+Today it's my great pleasure to announce the availability of Debezium *1.0.0.Final*!
+
+Since the https://github.com/debezium/debezium/commit/0a99ed67cd8f74d6f451b0a2d3809e23127e4698[initial commit] in November 2015,
+the Debezium community has worked tirelessly to realize the vision of building a comprehensive open-source low-latency platform for change data capture for a variety of databases.
+
+Within those four years, Debezium's feature set has tremendously grown: stable, highly configurable CDC connectors for MySQL, Postgres, MongoDB and SQL Server, incubating connectors for Apache Cassandra and Oracle, facilities for transforming and routing messages, support for design patterns such as the outbox pattern and much more.
+A very active, open and welcoming community of users, contributors and committers has formed around the project.
+Debezium is deployed to production at lots of organizations from all kinds of industries,
+partly with huge installations, using hundreds of connectors to stream data changes out of thousands of databases.
+
+The 1.0 release marks an important milestone for the project:
+based on all the production feedback we got from the users of the 0.x versions, we figured it's about time to express the maturity of the four stable connectors in the version number, too.
+
+== Why Debezium?
+
+One of the things making it so enjoyable to work on Debezium as a tool for change data capture is the variety of potential use cases.
+When presenting the project at conferences,
+it's just great to see how people quickly get excited when they realize all the possibilities enabled by Debezium and CDC.
```

Comment:
Define CDC?

Reply:
Good point. I've changed the first occurence in the post to "change data capture (CDC)".